scripts/content-types/load-content-types.ts

```diff
@@ -0,0 +1,252 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
+const fs = require('fs');
+const readline = require('readline');
+
+const JWP_HOST = process.env.JWP_HOST || 'https://api.jwplayer.com';
```

Comment:
Maybe we can also pass it as an argument? Could be useful to create schemas for dev

Reply:
I intentionally made this a less obvious param, because it's something only JWP devs will need to care about. You can always set it using:
`JWP_HOST=https://api-dev.jwplayer.com/ yarn load-content-types...`